karma.config.cjs

```diff
@@ -156,6 +156,11 @@ module.exports = (config) => {
         build: {
           target: 'esnext',
         },
+        server: {
+          warmup: {
+            clientFiles: ['src/**/*', 'test/**/*'],
```

Comment:
Adding the warmup fixes the issue with `useRef` hook that happens because of migration from `vite v5` to `Vite v6`